rdkit/Chem/MolDb/Loader_sa.py

```diff
@@ -68,7 +59,7 @@ def ProcessMol(session, mol, globalProps, nDone, nameProp='_Name', nameCol='comp
   except KeyError:
     nm = None
   if not nm:
-    nm = 'Mol_%d' % nDone
+    nm = f'Mol_{nDone}' % nDone
```

Comment:
The `% nDone` is redundant.

`nm = f'Mol_{nDone}'`

Reply:
I will fix it in the next patch

Reply:
please do